docker/image/whyis-server/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM maven:3.6-jdk-8 as build-java
```

Comment:
Not seeing the bundle .js build in here. It was merged into the uber-Dockerfile yesterday.
Please go ahead and build the bundle with a multi-stage nodejs build, then don't install nodejs into whyis-server-deps.